app/src/main/java/com/tks/trendsmeme/presentation/MainActivity.kt

```diff
@@ -24,35 +39,230 @@ import androidx.compose.runtime.remember
 import androidx.compose.runtime.setValue
 import androidx.compose.ui.Alignment
 import androidx.compose.ui.Modifier
+import androidx.compose.ui.draw.clip
+import androidx.compose.ui.graphics.Color
+import androidx.compose.ui.graphics.ColorFilter
 import androidx.compose.ui.platform.LocalContext
+import androidx.compose.ui.res.colorResource
 import androidx.compose.ui.res.painterResource
+import androidx.compose.ui.text.TextStyle
+import androidx.compose.ui.text.font.FontWeight
+import androidx.compose.ui.text.input.KeyboardType
 import androidx.compose.ui.text.input.PasswordVisualTransformation
 import androidx.compose.ui.text.input.TextFieldValue
 import androidx.compose.ui.tooling.preview.Preview
 import androidx.compose.ui.unit.dp
+import androidx.compose.ui.unit.sp
 import androidx.core.splashscreen.SplashScreen.Companion.installSplashScreen
 import com.tks.trendsmeme.R
 import com.tks.trendsmeme.extensions.showToast
+import com.tks.trendsmeme.presentation.customComponents.CustomComponents
+import com.tks.trendsmeme.presentation.pages.OtpScreen
 import com.tks.trendsmeme.ui.theme.TrendsMemeTheme
 import com.tks.trendsmeme.utils.SingleSelectDialog
 import androidx.compose.material3.Text as Text1
 
+
 class MainActivity : ComponentActivity() {
     override fun onCreate(savedInstanceState: Bundle?) {
+
         installSplashScreen().setKeepOnScreenCondition(
             condition = { false }
         )
         super.onCreate(savedInstanceState)
+
         enableEdgeToEdge()
         setContent {
             TrendsMemeTheme {
-                LoginScreen()
-                //MyScreen()
+                //EnterMobileNumberPage()
+                OtpScreen()
             }
         }
     }
 }
 
+@Composable
+fun EnterMobileNumberPage() {
+
+    Column(modifier = Modifier
+        .fillMaxSize()
+        .background(color = Color.Black),
+        verticalArrangement = Arrangement.Center, horizontalAlignment = Alignment.CenterHorizontally) {
+
+        AddAppLogo()
+        AddMobileEditText()
+        AddContinueButton()
+        SeparatorWithOrCircle()
+        SocialMediaLogin()
+        AddPrivacyText()
+    }
+
+
+
+
+
+}
+
+@Composable
+fun AddContinueButton() {
+
+    val context=LocalContext.current
+    val modifier = Modifier
+        .fillMaxWidth()
+        .height(80.dp)
+        .padding(16.dp)
+    CustomComponents().RoundedCornerButton(
+        onClick = { sendOtp(context) },
+        text = "Continue",
+        modifier
+    )
+}
+
+fun sendOtp(context: Context) {
+    "Otp sent to your mobile".showToast(context,Toast.LENGTH_LONG)
+}
```

Comment:
**Suggestion:** Consider adding error handling for the `sendOtp` function to manage cases where the OTP cannot be sent due to network issues or server errors. This will improve the robustness of the OTP functionality. [enhancement, importance: 9]
```suggestion
fun sendOtp(context: Context) {
    try {
        // Attempt to send OTP
        "Otp sent to your mobile".showToast(context,Toast.LENGTH_LONG)
    } catch (e: Exception) {
        "Failed to send OTP. Please try again.".showToast(context, Toast.LENGTH_LONG)
    }
}
```